access/grpc/convert_test.go

```diff
@@ -16,27 +16,28 @@
  * limitations under the License.
  */
 
-package convert_test
+package grpc_test
 
 import (
 	"testing"
 	"time"
 
+	"github.com/onflow/flow-go-sdk/access/grpc"
+
 	"github.com/onflow/cadence"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/onflow/flow-go-sdk"
-	"github.com/onflow/flow-go-sdk/client/convert"
 	"github.com/onflow/flow-go-sdk/test"
 )
 
 func TestConvert_Account(t *testing.T) {
 	accountA := test.AccountGenerator().New()
 
-	msg := convert.AccountToMessage(*accountA)
+	msg := grpc.AccountToMessage(*accountA)
```

Comment:
Do the convert functions still need to be exported from the `grpc` package? Would be good to make them private if they're not needed, just to reduce the package API surface area.